packages/@lightningjs/ui-components/src/components/MetadataBase/MetadataBase.js

```diff
@@ -33,41 +33,23 @@ class MetadataBase extends Base {
   }
 
   static _template() {
+    // Comments indicate where child components are inserted
     return {
       Text: {
         flex: { direction: 'column', justifyContent: 'flex-start' },
-        Title: {
-          type: TextBox,
-          signals: {
-            textBoxChanged: '_titleLoaded'
-          }
-        },
+        // Title: {},
         DetailsWrapper: {
-          Details: {
-            type: TextBox,
-            signals: {
-              textBoxChanged: '_detailsLoaded'
-            }
-          }
-        },
-        Description: {
-          type: TextBox,
-          signals: {
-            textBoxChanged: '_descriptionLoaded'
-          }
+          // Details: {}
         }
-      },
-      Logo: {
-        flexItem: false,
-        type: Icon
+        // Description: {}
       }
+      // Logo: {
+      //   flexItem: false,
+      //   type: Icon
+      // }
     };
   }
 
-  _init() {
-    this.requestUpdate(true);
```

Comment:
This did not seem necessary, but if anyone knows the origins?

Reply:
🤔 seems like very old leftovers to me. @ImCoolNowRight any thoughts? If everything seems to render as expected without it in Tile and Cards, it's probably OK

Reply:
My best guess is that this was from some weird formatting bug that could only be solved with an extra update cycle. It's looking fine now, so it probably got resolved and this just got left in because I didn't make a note of when to remove it.